README.md

```diff
@@ -328,66 +349,165 @@ The complete list of transformations supported and their usage in ImageKit can b
+| overlayHeight                 | overlayHeight(overlayHeight: Int)                                                                                                                                                                                                                                                                                                                    | oh                      |
+| overlayWidth                  | overlayWidth(overlayWidth: Int)                                                                                                                                                                                                                                                                                                                      | ow                      |
+| overlayImage                  | overlayImage(overlayImage: String)                                                                                                                                                                                                                                                                                                                   | oi                      |
+| overlayImageTrim              | overlayImageTrim(overlayImageTrim: Boolean)                                                                                                                                                                                                                                                                                                          | oit                     |
+| overlayImageAspectRatio       | overlayImageAspectRatio(width: Int, height: Int)                                                                                                                                                                                                                                                                                                     | oiar                    |
+| overlayImageBackground        | overlayImageBackground(overlayImageBackground: String)                                                                                                                                                                                                                                                                                               | oibg                    |
+| overlayImageBorder            | overlayImageBorder(borderWidth: Int, borderColor: String)                                                                                                                                                                                                                                                                                            | oib                     |
```

Comment:
Please remove all `overlay` related parameter. In future, all these parameters will be removed and new powerful layer syntax will be the only way to use overlays. In SDKs, we will expose `raw` parameter and allow users to set any string.

Reply:
Done